backend/src/models/fhir_immunization_post_validators.py

```diff
@@ -24,72 +25,87 @@ class PostValidators:
     """FHIR Immunization Post Validators"""
     def __init__(self, immunization):
         self.values = immunization
+        self.errors = []
     
     def validate(self):
         """
         Run all post-validation checks.
         """
+        
+        #Run critical validations that other validations rely on, if they fail then no need to carry on
         try:
             self.validate_and_set_vaccination_procedure_code(self.values)
-            self.set_status(self.values)
```

Comment:
Does `status` need to be in the critical validations? I know a lot of the validators are dependent on status.

Reply:
never mind I'm an idiot, it's set further down.